app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<h2>Sign up</h2>
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+  <div class="field">
+    <%= f.label :UserName %><br />
```

Comment:
- [ ] This label should be `:email`, not `:UserName` as it was before.

Reply:
thanks alot resoved